src/index.js

```diff
@@ -57,6 +63,7 @@ function setup(webContents, { socketTimeout, socketKeepAliveDelay } = {}) {
       );
       // Notify the renderer process that we are listening for notifications
       webContents.send(NOTIFICATION_SERVICE_STARTED, credentials.fcm.token);
+      started = true;
```

Comment:
Nit: moving `start` later means that if we get two `START_NOTIFICATION_SERVICE` events, we might have two concurrent checkin and registration calls mid-flight. 

This is prolly good enough, but if we wanted to be "more correct" we could probably cache a promise instead of a boolean. 
in english - if we have no registration promise, start a new one and cache a promise for it. if we already have a cached registration promise, return it.